client/src/components/Festivals.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+import styled from '@emotion/styled';
+import Button from './Button';
+
+const Festival = styled.div`
+  padding-top: 210px;
+  text-align: center;
+  font-size: 34px;
+  font-family: 'Quicksand Book';
+  margin-bottom: 20px;
+  text-transform: uppercase;
+`;
+
+const FestivalDetail = styled.div`
+  text-align: center;
+  font-size: 20px;
+  font-family: 'Quicksand Book';
+  margin-bottom: 30px;
+  text-transform: uppercase;
+`;
+
+const FestivalIntroduction = styled.div`
+  max-width: 600px;
+  font-size: 12px;
+  font-family: 'Quicksand Light';
+  line-height: 1.3;
+  margin-bottom: 30px;
+  text-transform: uppercase;
+`;
+
+const LineUp = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 20px;
+  margin-bottom: 20px;
+  font-size: 12px;
+  font-family: 'Quicksand Book';
+  text-transform: uppercase;
+`;
+
```

Comment:
I'm wondering why you are just working with divs and don't use h1, h2, p, etc.?

Reply:
good question actually. I think there's a need to refactor anyways, so I put that on my todo list